src/Symfony/Component/Form/Test/FormPerformanceTestCase.php

```diff
@@ -25,19 +25,25 @@ abstract class FormPerformanceTestCase extends FormIntegrationTestCase
 {
     use VersionAwareTest;
 
+    private float $startTime;
     protected int $maxRunningTime = 0;
 
-    protected function runTest(): mixed
+    protected function setUp(): void
     {
-        $s = microtime(true);
-        $result = parent::runTest();
-        $time = microtime(true) - $s;
+        parent::setUp();
+
+        $this->startTime = microtime(true);
+    }
+
+    protected function assertPostConditions(): void
+    {
+        parent::assertPreConditions();
```

Comment:
calling the parent method is good, but only if you call the parent method with the same name

Reply:
oh wow 🙈